neuralmonkey/learning_utils.py

```diff
@@ -396,8 +396,8 @@ def print_final_evaluation(name: str, eval_result: Evaluation) -> None:
     line_len = 22
     log("Evaluating model on \"{}\"".format(name))
 
-    for name, value in eval_result.items():
-        space = "".join([" " for _ in range(line_len - len(name))])
+    for i_name, value in eval_result.items():
+        space = "".join([" " for _ in range(line_len - len(i_name))])
```

Comment:
Nevim, proč zrovna `i_`, ale přejmenované je to protože si pylint stěžoval, že to overituje nějaký jiný name.